source/renderer/app/components/wallet/WalletRestoreDialog.js

```diff
@@ -486,10 +569,7 @@ export default class WalletRestoreDialog extends Component<Props, State> {
           }
           placeholder={
             !this.isCertificate()
-              ? intl.formatMessage(messages.recoveryPhraseInputHint, {
-                  numberOfWords:
-                    walletType === WALLET_RESTORE_TYPES.LEGACY ? '12' : '15',
```

Comment:
@DeeJayElly we need to keep this.

Reply:
We don't need it for placeholder as the placeholder is changed now in the new design https://app.zeplin.io/project/57acfaa7436a750316161a8b/screen/5de4e5372efdcb7bb478f975